client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/TemperatureChart.tsx

```diff
@@ -95,6 +97,62 @@ const Chart = ({
     return <NothingHere body={t('error.no-temperature-logs')} />;
   }
 
+  const tickSpace =
+    (yAxisDomain[1] - yAxisDomain[0]) / (NUMBER_OF_HORIZONTAL_LINES + 1);
+  const ticks = Array.from({ length: NUMBER_OF_HORIZONTAL_LINES }).map(
+    (_, index) => (index + 1) * tickSpace
+  );
+  ticks.push(yAxisDomain[0]);
+  ticks.push(2);
+  ticks.push(8);
+  ticks.push(yAxisDomain[1]);
+  ticks.sort((a, b) => (a > b ? 1 : -1));
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const CustomisedTick = ({ x, y, payload }: any) => {
+    const theme = useTheme();
+    const textColour =
+      payload.value === 2
+        ? theme.palette.chart.cold.main
+        : payload.value === 8
+        ? theme.palette.chart.hot.main
+        : theme.palette.gray.dark;
+    return (
+      <g>
+        <line x={x} y={y} stroke={theme.palette.gray.dark}></line>
+        <text
+          x={x}
+          y={y}
+          fill={textColour}
+          textAnchor="end"
+          style={{
+            fontSize: 12,
+            fontWeight:
+              payload.value === 2 || payload.value === 8 ? 'bold' : '',
```

Comment:
Might be a good idea to use a constant to explain these values....
`LOWER_THRESHOLD` and `UPPER_THRESHOLD`

Reply:
Not actually a huge fan of constants as they can be overused, but when I see unexplained numbers in 2 places I think that could become a point of confusion in future...

Reply:
I see they are on lines 115 and 117 too.